src/components/Publish/index.tsx

```diff
@@ -279,10 +280,15 @@ export default function PublishPage({
     >
       {({ values }) => (
         <>
-          <PageHeader
-            title={<Title networkId={values.user.chainId} />}
-            description={content.description}
-          />
+          <div className={styles.header}>
+            <PageHeader
+              title={<Title networkId={values.user.chainId} />}
+              description={content.description}
+            />
+          </div>
+          <div style={{ marginTop: '2rem', marginBottom: '2rem' }}>
+            <Wallet />
```

Comment:
the button says CONNECTWALLET, should that not be CONNECT WALLET? not sure where the issue comes from

Reply:
It comes from the original code. The text is correct, the display is wrong. Fixed.